Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -831,6 +782,48 @@ namespace UnitTest
         EXPECT_NEAR(newCapsuleHeight, 1.2f, ManipulatorTolerance);
     }
 
+    TEST_F(
+        PhysXEditorColliderComponentManipulatorFixture,
+        CapsuleColliderAsymmetricalScaleManipulatorsCorrectlyLocatedRelativeToColliderWithNonUniformScale)
+    {
+        const float capsuleRadius = 0.2f;
+        const float capsuleHeight = 1.0f;
+        const AZ::Quaternion capsuleRotation(-0.2f, -0.8f, -0.4f, 0.4f);
+        const AZ::Vector3 capsuleOffset(1.0f, -2.0f, 1.0f);
+        SetupCollider(Physics::CapsuleShapeConfiguration(capsuleHeight, capsuleRadius), capsuleRotation, capsuleOffset);
+        const AZ::Quaternion entityRotation(0.7f, -0.1f, -0.1f, 0.7f);
+        const AZ::Vector3 entityTranslation(-2.0f, 1.0f, -3.0f);
+        const float uniformScale = 2.0f;
+        SetupTransform(entityRotation, entityTranslation, uniformScale);
+        const AZ::Vector3 nonUniformScale(1.0f, 0.5f, 1.5f);
+        SetupNonUniformScale(nonUniformScale);
+        EnterColliderSubMode(PhysX::ColliderComponentModeRequests::SubMode::Dimensions);
+
+        // the expected position of the collider centre based on the combination of entity transform, collider offset and non-uniform scale
```

Comment:
uber nit: I know you'll hate me for mentioning this but we're supposed to use 'American English' in the codebase so this should be 'center' 😝 (repeated)

Reply:
Changed to "central point" to avoid hating myself.

Reply:
:us:

Reply:
😂 brilliant, good compromise 😅